### PR TITLE
Updated the Import_Data_Task and Fetch_Images_From_Satellite_Platform…

### PR DESCRIPTION
**Pull request by carolinepacheco**
…s sections.

### CODE REVIEW

File: src/docs/PML/PMLUserGuide.adoc [hunk 1]
```diff
@@ -3459,7 +3465,7 @@ NOTE: More information about the source of this task can be found https://scikit
 | Specifies the parameters' values of the AdaBoost algorithm. Check the list of parameters https://scikit-learn.org/stable/modules/generated/sklearn.cluster.KMeans.html[here^].
 | JSON format
 | `TYPE`
-| Enter the type of algorithm. There are two possible: classification or regression.
+| Enters the type of algorithm. There are two possible: classification or regression.
```
Comment (ImenBizid) [inline src/docs/PML/PMLUserGuide.adoc:3468]: Specifies the type of algorithm (classification or regression);

File: src/docs/PML/PMLUserGuide.adoc [hunk 2]
```diff
@@ -3493,7 +3499,7 @@ NOTE: More information about the source of this task can be found https://scikit
 | Specifies the parameters' values of the CatBoost algorithm. Check the list of parameters https://github.com/catboost[here^].
 | JSON format
 | `TYPE`
-| Enter the type of algorithm. There are two possible: classification or regression.
+| Enters the type of algorithm. There are two possible: classification or regression.
```
Comment (ImenBizid) [inline src/docs/PML/PMLUserGuide.adoc:3502]: same here

File: src/docs/PML/PMLUserGuide.adoc [hunk 3]
```diff
@@ -2589,15 +2589,21 @@ NOTE: More information about this dataset can be found https://scikit-learn.org/
...
 | String
 | `FILE_DELIMITER`
-| Delimiter to use.
+| Defines a delimiter to use.
+| String (default=;)
+| `LABEL_COLUMN`
+| Enters the column label name.
 | String
+| `LIMIT_OUTPUT_VIEW`
+|  Specifies how many rows of the dataframe will be previewed in the browser to check each task results.
```
Comment (mboussaa) [inline src/docs/PML/PMLUserGuide.adoc:2605]: there is an extra space before specifies

File: src/docs/PML/PMLUserGuide.adoc [hunk 4]
```diff
@@ -3526,7 +3532,7 @@ NOTE: More information about the source of this task can be found https://scikit
 | Specifies the parameters' values of the Gradient Boosting algorithm. Check the list of parameters https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.GradientBoostingClassifier.html[here^].
 | JSON format
 | `TYPE`
-| Enter the type of algorithm. There are two possible: classification or regression.
+| Enters the type of algorithm. There are two possible: classification or regression.
```
Comment (ImenBizid) [inline src/docs/PML/PMLUserGuide.adoc:3535]: same here

File: src/docs/PML/PMLUserGuide.adoc [hunk 5]
```diff
@@ -3559,7 +3565,7 @@ NOTE: More information about the source of this task can be found https://scikit
 | Specifies the parameters' values of the Random Forest algorithm. Check the list of parameters https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestClassifier.html[here^].
 | JSON format
 | `TYPE`
-| Enter the type of algorithm. There are two possible: classification or regression.
+| Enters the type of algorithm. There are two possible: classification or regression.
```
Comment (ImenBizid) [inline src/docs/PML/PMLUserGuide.adoc:3568]: same here

File: src/docs/PML/PMLUserGuide.adoc [hunk 6]
```diff
@@ -3593,7 +3599,7 @@ NOTE: More information about the source of this task can be found https://scikit
 | Specifies the parameters' values of the XGBoost algorithm. Check the list of parameters https://xgboost.readthedocs.io/en/latest/python/python_api.html[here^].
 | JSON format
 | `TYPE`
-| Enter the type of algorithm. There are two possible: classification or regression.
+| Enters the type of algorithm. There are two possible: classification or regression.
```
Comment (ImenBizid) [inline src/docs/PML/PMLUserGuide.adoc:3602]: same here

File: src/docs/PML/PMLUserGuide.adoc [hunk 7]
```diff
@@ -3655,10 +3661,10 @@ NOTE: More information about the source of this task can be found https://scikit
 | If False, the task will be ignored, it will not be executed.
 | Boolean (default=True)
 | `LABEL_COLUMN`
-| Enter the column label name.
+| Enters the column label name.
```
Comment (ImenBizid) [inline src/docs/PML/PMLUserGuide.adoc:3664]: same

File: src/docs/PML/PMLUserGuide.adoc [hunk 8]
```diff
@@ -3624,10 +3630,10 @@ NOTE: More information about the source of this task can be found https://scikit
...
 | String
 | `USE_NVIDIA_RAPIDS`
-| Enable NVIDIA RAPIDS support.
+| Enables NVIDIA RAPIDS support.
```
Comment (ImenBizid) [inline src/docs/PML/PMLUserGuide.adoc:3636]: Refers to the column "label name"

File: src/docs/PML/PMLUserGuide.adoc [hunk 9]
```diff
@@ -3677,7 +3683,7 @@ NOTE: More information about the source of this task can be found https://scikit
 | If False, the task will be ignored, it will not be executed.
 | Boolean (default=True)
 | `LABEL_COLUMN`
-| Enter the column label name.
+| Enters the column label name.
```
Comment (ImenBizid) [inline src/docs/PML/PMLUserGuide.adoc:3686]: same

File: src/docs/PML/PMLUserGuide.adoc [hunk 10]
```diff
@@ -3686,7 +3692,7 @@ NOTE: More information about the source of this task can be found https://scikit
 | 2D Partial Dependence Plots show predictions for any combination of two features.
 | String [e.g, distance_circularity, max_length_aspect_ratio].
 | `SHAP_ROW_SHOW`
-| Enter the row of data to show. The SHAP (https://github.com/slundberg/shap) values interpret the impact of having a certain value for a given feature in comparison to the prediction we'd make if that feature took some baseline value. Feature values causing increased predictions are in pink and  Feature values decreasing the prediction are in blue. The SHAP_ROW_SHOW The row of data to show
+| Defines the row of data to show. The SHAP (https://github.com/slundberg/shap) values interpret the impact of having a certain value for a given feature in comparison to the prediction we'd make if that feature took some baseline value. Feature values causing increased predictions are in pink and  Feature values decreasing the prediction are in blue. The SHAP_ROW_SHOW The row of data to show
```
Comment (mboussaa) [inline src/docs/PML/PMLUserGuide.adoc:3695]: we'd =? we would

Comment (mboussaa) [reply src/docs/PML/PMLUserGuide.adoc:3695]: extra space before  Feature values

File: src/docs/PML/PMLUserGuide.adoc [hunk 11]
```diff
@@ -4716,16 +4722,19 @@ image::ciar_diagram.png[align=center]
...
+| List [PA:URL,PA:URI,PA:USER_FILE,PA:GLOBAL_FILE] (default=PA:GLOBAL_FILE)
+| `FILE_PATH`
+| Inserts a file path/name.
+= | String
 | `TIME_TO_RETRIEVE_IN_SECONDS`
 | Defines time in second to request an offline product from the Long Term Archive (LTA) in the Copernicus Open Access Hub’s.
 | Int (default=900 seconds)
 | `TIME_TO_CHECK_ONLINE_IN_SECONDS`
-| Defines time in second to check if a product is online.
+| Determines time in second to check if a product is online.
```
Comment (mboussaa) [inline src/docs/PML/PMLUserGuide.adoc:4734]: I prefer defines and defines **the** time in second**s**

File: src/docs/PML/PMLUserGuide.adoc [hunk 12]
```diff
@@ -4741,7 +4750,8 @@ NOTE: Please click https://peps.cnes.fr/rocket/#/register[here^] to create a new
 
 NOTE: Please click https://scihub.copernicus.eu/dhus/#/self-registration[here^] to create a new user account from Copernicus website.
 
-WARNING: Note que as variaveis `TIME_TO_RETRIEVE_IN_SECONDS`, `TIME_TO_CHECK_ONLINE_IN_SECONDS` and `WALLTIME` are valid only for the Copernicus platform.
+WARNING: Note que as variaveis `TIME_TO_RETRIEVE_IN_SECONDS`, `TIME_TO_CHECK_ONLINE_IN_SECONDS` and `WALLTIME` are valid only for the Copernicus platform. In addition, it is necessary to define a policy used by the ProActive Scheduler, in this case only two tasks must be executed simultaneously.
```
Comment (mboussaa) [inline src/docs/PML/PMLUserGuide.adoc:4753]: valid only => only valid

Comment (mboussaa) [reply src/docs/PML/PMLUserGuide.adoc:4753]: , in this case only two tasks must be executed simultaneously. => . In this case, only two tasks must be executed simultaneously.

**Review by mboussaa — APPROVED** [review]

**Review by andrewssobral — APPROVED** [review]

File: src/docs/PML/PMLUserGuide.adoc [hunk 13]
```diff
@@ -4741,7 +4754,8 @@ NOTE: Please click https://peps.cnes.fr/rocket/#/register[here^] to create a new
 
 NOTE: Please click https://scihub.copernicus.eu/dhus/#/self-registration[here^] to create a new user account from Copernicus website.
 
-WARNING: Note que as variaveis `TIME_TO_RETRIEVE_IN_SECONDS`, `TIME_TO_CHECK_ONLINE_IN_SECONDS` and `WALLTIME` are valid only for the Copernicus platform.
+WARNING: Note que as variaveis `TIME_TO_RETRIEVE_IN_SECONDS`, `TIME_TO_CHECK_ONLINE_IN_SECONDS` and `WALLTIME` are only valid for the Copernicus platform. In addition, it is necessary to define a policy used by the ProActive Scheduler. In this case, only two tasks must be executed simultaneously.
+
```
Comment (Hiba-Alili) [inline src/docs/PML/PMLUserGuide.adoc:4758]: Do you mean, Note that the variables?

**Review by Hiba-Alili — APPROVED** [review]
